lib/api/src/externals/function.rs

```diff
@@ -1285,6 +1167,7 @@ mod inner {
 
     /// An empty struct to help Rust typing to determine
     /// when a `HostFunction` does not have an environment.
+    #[derive(Clone)]
```

Comment:
Cloning an empty struct? Why? Is the struct just used in the type system? Or is it compared by pointer?

Reply:
It's required for `WasmerEnv` to be clonable